### PR TITLE
Hides update button of profile page for virtual accounts

### PR DESCRIPTION
**Pull request by morteza-binary**

### CODE REVIEW

File: www/js/pages/profile/profile.template.html [hunk 1]
```diff
@@ -316,7 +316,10 @@ <h1 class="page-title">
             <div class="row">
                 <div class="col center">
                     <span class="help-block error center" ng-if="vm.notAnyChanges">{{'financial-assessment.no_change' | translate}}</span>
-                    <button class="button button-block" ng-click="vm.updateProfile()" ng-disabled="!profileForm.$valid || vm.disableUpdateButton  || (vm.isFinancial && vm.profile.tax_residence  == null) ">
+                    <button class="button button-block"
```
Comment (nazaninreihani) [inline www/js/pages/profile/profile.template.html]: We have some cases in which the user has not set the country of residence at all and he/she gets a notification to choose the country and we redirect him/her to update profile page, we have to have update button enabled for that case

Comment (morteza-binary) [inline www/js/pages/profile/profile.template.html]: @nazaninreihani But Ashkan told that profile page for virtual account is read only. let me know how can create a virtual account without country of residence. I don't think there is a method to create a user without residence country.

**Comment by ashkanx** [comment]
It use to be like that. So we should handle old accounts.

Ashkan
mobile

**Comment by morteza-binary** [comment]
@ashkanx you meant there are accounts that don't have residence country value?

**Comment by ashkanx** [comment]
Yes. Correct.


Ashkan
mobile

**Comment by morteza-binary** [comment]
So we should revert it again 😄

**Comment by nazaninreihani** [comment]
We can hide update button when the country exists, do you want me to do this change?

**Comment by morteza-binary** [comment]
yes we can do it, I'll be glad if you do it :-)

**Pull request merged**